src/detections.c

```diff
@@ -0,0 +1,16 @@
+#include "detections.h"
+#include <libopencm3/stm32/gpio.h>
+#include <libopencm3/stm32/timer.h>
+/**
+ * @brief TIM1 interruption routine.
```

Comment:
Maybe a space (or 2) between the includes and the docstring?

Reply:
The clang-format formatting changes from 2 spaces to 1,but it is not working from no spaces to one.